src/librustc_typeck/diagnostics.rs

```diff
@@ -934,6 +934,28 @@ The number of supplied parameters much exactly match the number of defined type
 parameters.
 "##,
 
+E0088: r##"
+You gave too many lifetimes parameters. Erroneous code example:
+
+```
+fn f() {}
```

Comment:
I'll add this note. However, I didn't find out yet out to "force pass" lifetime parameter to a function.